configure

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
```

Comment:
> I think this should be python2. See apiaryio/snowcrash#277

As you have correctly assumed. Hardcoding python2 is not the correct solution. Does not work on systems with Python 2.7.6 only.